pom.xml

```diff
@@ -89,7 +89,7 @@
       <dependency>
         <groupId>org.robolectric</groupId>
         <artifactId>android-all</artifactId>
-        <version>5.0.0_r2-robolectric-0</version>
+        <version>5.0.0_r2-robolectric-1</version>
```

Comment:
Yes, there are a lot of identical classes in layoutlib.jar which are essentially modified versions of those in android-all.jar. For example, almost all widgets like TextView, ImageView, etc. have another version in layoutlib.jar. As stated in the comments of layoutlib source files, they are provided as a version which cannot be run on real devices but able to be rendered in simulation. I am looking into loading class from either layoutlib or android-all according to the rendering flag.